sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -603,6 +687,15 @@ def __repr__(self):
             :1024
         ]
 
+    def to_dict(self):
+        return {
+            "page_number": self.page_number,
+            "row_count": self.row_count,
+            "column_count": self.column_count,
+            "cells": [cell.to_dict() for cell in self.cells],
+            "bounding_box": [box.to_dict() for box in self.bounding_box]
```

Comment:
for v2.0 compatibility, we should also do a None check (my bad)

```suggestion
            "bounding_box": [box.to_dict() for box in self.bounding_box] if self.bounding_box else []
```

Reply:
Done